openedx/core/openapi.py

```diff
@@ -44,7 +44,7 @@ def dedent(text):
     return textwrap.dedent(text)
 
 
-def swagger_auto_schema(**kwargs):
+def swagger_auto_schema(*args, **kwargs):
```

Comment:
As this function gets smarter, should we name it something different in order to clarify that it isn't the same thing as `drf_yasg.utils.swagger_auto_schema`? @nedbat

Reply:
`@swaggerify`?

Reply:
Yes, it should have a different name.  In my pull request, the decorator behaved the same as drf_yasg's except for some indentation tweaks.  Now it's different.

I don't quite understand where people say "swagger" and where they say "openapi"?

Reply:
A perhaps minor point: you added `*args` here, but we shouldn't allow positional args for this decorator.  Requiring everything to be named will be much better.

Reply:
> Lets at least make it an error to provide *args when they aren't arguments for the decorator.

Agreed, I implemented a ValueError.

Reply:
I don't quite understand the difference between swagger and openapi, so I decided to rename the decorator to `document_api_view`. I'm open to alternative propositions.